platform/default/src/mbgl/storage/offline_database.cpp

```diff
@@ -606,6 +606,47 @@ bool OfflineDatabase::putTile(const Resource::TileData& tile,
     return true;
 }
 
+std::exception_ptr OfflineDatabase::invalidateTileCache() try {
+    // clang-format off
+    mapbox::sqlite::Query query{ getStatement(
+        "UPDATE tiles "
```

Comment:
I see that same pattern is applied everywhere in the file, but is it fine that we hard-code all the table and field names in the queries?

Reply:
IMO it is fine, the field names won't change.